config/locales/static/accessibility_statement.yml

```diff
@@ -2,17 +2,83 @@
+        as possible to be able to use this website. For example, that means you
+        should be able to:
 
-        * the efforts we have made to comply with the Public Sector Bodies
-        (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018
-        * how this website meets the Web Content Accessibility Guidelines version
-        2.1 AA standard
+        * change colours, contrast levels and fonts
+        * zoom in up to 300% without the text spilling off the screen
+        * navigate most of the website using just a keyboard
+        * navigate most of the website using speech recognition software
+        * listen to most of the website using a screen reader (including the most
+          recent versions of JAWS, NVDA and VoiceOver)
 
-        If you have any feedback on the accessibility of this website, please
-        [contact us](mailto:child-development.training@education.gov.uk?subject=Accessibility){:class="govuk-link"}.
+        We've also made the website text as simple as possible to understand.
+
+        <a href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopner noreferrer">
+        AbilityNet (opens in a new tab)</a> has advice on making your device
+        easier to use if you have a disability.
+
+        ## How accessible this website is
+
+        We know some parts of this website are not fully accessible:
+
+        * if you disable javascript in your browser, the alert to tell you that you will soon be signed out due to inactivity may not appear as accessibly as the version displayed when you have javascript enabled
+        * our sitemap includes only the pages that do not need you to be signed in - this is because once you have started a module, you will need to work through each page in order
+
+        ## Feedback and contact information
+
+        If you need information on this website in a different format like
+        accessible PDF, large print, easy read, audio recording or braille, email <a href="mailto:child-development.training@education.gov.uk">child-development.training@education.gov.uk</a>.
+
+        We'll consider your request and get back to you in 10 days.
+
+        ## Reporting accessibility problems with this website
+
+        We're always looking to improve the accessibility of this website. If you
+        find any problems not listed on this page or think we're not meeting
+        accessibility requirements, email <a href="mailto:child-development.training@education.gov.uk">child-development.training@education.gov.uk</a>.
+
+        ## Enforcement procedure
+
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing
+        the Public Sector Bodies (Websites and Mobile Applications) (No. 2)
+        Accessibility Regulations 2018 (the 'accessibility regulations'). If you're
+        not happy with how we respond to your complaint, <a rel="noopner noreferrer" href="https://www.equalityadvisoryservice.com/" target="_blank">
```

Comment:
```suggestion
        not happy with how we respond to your complaint, <a rel="noopener noreferrer" href="https://www.equalityadvisoryservice.com/" target="_blank">
```

Looks like a minor typo

Reply:
Ah that's from the template! I'll fix and let the team know